processing/src/main/java/io/druid/segment/data/ColumnarLongs.java

```diff
@@ -39,6 +40,39 @@
   @Override
   void close();
 
+  default ColumnValueSelector<Long> makeColumnValueSelector(ReadableOffset offset, ImmutableBitmap nullValueBitmap)
```

Comment:
Align logic and method order with double and float.

Reply:
done.